sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -202,10 +206,7 @@
         PROPERTY_AZURE_REQUEST_READ_TIMEOUT
     };
 
-    /*
-     * Gets the global configuration shared by all client libraries.
-     */
-    private static final Configuration GLOBAL_CONFIGURATION = new Configuration();
+    public static final ConfigurationSource[] CONFIG_SOURCES = new ConfigurationSource[]{ConfigurationSource.ENV_VAR_SOURCE, ConfigurationSource.SYSTEM_PROP_SOURCE};
```

Comment:
nit: This has changed the default load ordering, before it was system properties then environment variables